frontend/components/Documents.js

```diff
@@ -43,6 +110,35 @@ const Documents = () => {
             <p>
                 This page displays all the documents stored in backend.
             </p>
+            <div>
+                <form onSubmit={handleSubmit}>
+                    <div className="form-group">
+                        <label htmlFor="author">Author</label>
+                        <input type="text" className="form-control"
+                            id="author" value={newDocData.author}
+                            onChange={handleAuthorInputChange}/>
+                    </div>
+                    <div className="form-group">
+                        <label htmlFor="title">Title</label>
+                        <input type="text" className="form-control"
+                            id="title" value={newDocData.title}
+                            onChange={handleTitleInputChange}/>
+                    </div>
+                    <div className="form-group">
+                        <label htmlFor="date">Date</label>
+                        <input type="number" className="form-control"
+                            id="date" value={newDocData.date}
+                            onChange={handleDateInputChange}/>
+                    </div>
+                    <div className="form-group">
+                        <label htmlFor="text">Text</label>
+                        <textarea className="form-control" id="text"
+                            rows="8" value={newDocData.text}
+                            onChange={handleTextInputChange}></textarea>
+                    </div>
+                    <button type="submit">Add</button>
+                </form>
+            </div>
```

Comment:
The most nitpicky of nitpicks, but we can get rid of the `<div>` that's wrapping this `<form>`, unless you have a specific design purpose in mind. Both `<div>`s and `<form>`s are block-level elements by default, so the `<div>` doesn't offer anything from a semantic or styling purpose yet. In general, it's very easy to write more markup than we need and clutter up the DOM, so I'll call out places that don't necessarily _need_ to exist. If you've got some design purpose in mind, though, and intend to add a class to the `<div>` or something, feel free to ignore me!

Reply:
updated!